### ISSUE TITLE
Zalenium failed to start after 2 minutes, failing

### ISSUE THREAD

**Issue by gary-a-zhang** [zalando/zalenium#1082]
## 🐛 Bug Report

Zalenium is not able to start. 

## To Reproduce

qa@VMUBUNTU1804QA01:~/eMite$ curl -sSL https://raw.githubusercontent.com/dosel/t/i/p | bash -s start
Checking dependencies... Done Checking dependencies.
latest: Pulling from dosel/zalenium
Digest: sha256:5f3d7cb213ac941d8880ebeaba6d74cf721987dde94e1ab5536ac9ada12e2c32
Status: Image is up to date for dosel/zalenium:latest
docker.io/dosel/zalenium:latest
latest: Pulling from elgalu/selenium
Digest: sha256:79995e14ebf832734a8bee5813e817d406c3825dba1cf0cdab3367f7b1f4acf0
Status: Image is up to date for elgalu/selenium:latest
docker.io/elgalu/selenium:latest
Starting Zalenium in docker...
WARN: Sauce Labs will not be enabled because the var $SAUCE_USERNAME is NOT present
WARN: BrowserStack will not be enabled because the var $BROWSER_STACK_USER is NOT present
WARN: Testing Bot will not be enabled because the var $TESTINGBOT_SECRET is NOT present
e60c7d006fc3532b5896b0bac3dadaf9c57df8e388abb6e111b93f8e5b8ea29d
................................................................................................................................................................................................................................................................................................................................................................................................................................................................Zalenium failed to start after 2 minutes, failing...
Docker binary already present, will use that one.
usermod: no changes
stat: /var/run/docker.sock:: 999
sudo groupadd --gid 999 docker:: docker
Adding user seluser to group docker
-- LOG 01:47:09:025260961 Ensuring docker works...
-- LOG 01:47:09:418101122 Ensuring docker-selenium is available...
haveged: haveged starting up
Copying files for Dashboard...

## Expected behavior

Zalenium can start.

## Environment

OS: Linux VMUBUNTU1804QA01 5.0.0-32-generic #34~18.04.2-Ubuntu SMP Thu Oct 10 10:36:02 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
Zalenium Image Version(s): latest(3.141.59w)
Docker version: 19.03.5, build 9013bf583a
Docker-Compose version (if applicable):
Exact Docker command to start the containers (if using docker-compose, provide 
the docker-compose file as well):
If using Kubernetes, specify your environment, and if relevant your manifests: NA

## Installation steps of Zalenium

```
sudo apt-get update
sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
sudo apt-get install docker-ce docker-ce-cli containerd.io
sudo docker run hello-world

sudo docker pull elgalu/selenium
sudo docker pull dosel/zalenium
sudo usermod -aG docker $USER
sudo reboot -f
sudo apt-get install jq
curl -sSL https://raw.githubusercontent.com/dosel/t/i/p | bash -s start
```

## Troubleshooting
I believe it's stuck at "sudo chown -R ${HOST_UID}:${HOST_GID} /home/seluser" of zalenium.sh. I also raised a stackoverflow [ticket](https://stackoverflow.com/questions/59208068/execution-of-chowncontainer-files-get-hung). I followed same steps on aws server which has same linux version. There is no problem.

```
qa@VMUBUNTU1804QA01:~$  docker exec -ti zalenium /bin/bash
bash: /home/seluser/.bashrc: Operation not permitted
seluser@e60c7d006fc3:~$ ps -ef
UID        PID  PPID  C STIME TTY          TIME CMD
root         1     0  0 01:47 pts/0    00:00:00 sudo --preserve-env gosu seluser ./zalenium.sh start --timeZone Australia/Sydney --desiredContainers 2 --maxDockerSelenseluser     31     1  0 01:47 pts/0    00:00:00 bash ./zalenium.sh start --timeZone Australia/Sydney --desiredContainers 2 --maxDockerSeleniumContainers 60 --screenWidroot       123     1  0 01:47 ?        00:00:00 haveged
root       128    31  0 01:47 pts/0    00:00:00 sudo chown -R 1000:1000 /home/seluser
root       129   128  0 01:47 pts/0    00:00:00 chown -R 1000:1000 /home/seluser
seluser    130     0  0 01:48 pts/1    00:00:00 /bin/bash
seluser    136   130  0 01:48 pts/1    00:00:00 ps -ef
seluser@e60c7d006fc3:~$
```

**Comment by gary-a-zhang** [comment]
After reinstalling the docker and zalenium with non-root user, there is some progress.

**Comment by gary-a-zhang** [comment]
There is still a problem.

**Comment by gary-a-zhang** [comment]
Hello, any finding? I guess it's stuck due to the server configuration, but I don't know exactly where and why as there is not enough output log. Any suggestion?

**Comment by gary-a-zhang** [comment]
Just find nginx is not started. Not sure if this will help.

/# ps -ef|grep nginx
root     28573 27442  0 15:15 pts/1    00:00:00 grep --color=auto nginx

**Comment by gary-a-zhang** [comment]
Update the issue and reopen the issue as I couldn't find the cause. It should be zalenium or our environment. After around one month investigation, still not able to solve the problem. Could you give me some suggestion?

**Comment by diemol** [comment]
Since this is a question rather than an issue, I already started replying in the StackOverflow issue linked above.
I'll close this issue and continue the conversation over there.

**Comment by atsai1220** [comment]
Ran into the same issue.
We had to delete our nfs volumes for `zalenium-data` and `zalenium-videos` and reinstall Zalenium so there aren't as many files to `chown`. We used Helm to install Zalenium in the first place so it was straight forward.

Here is the Helm v3 Chart: https://raw.githubusercontent.com/zalando/zalenium/master/charts/zalenium


Then we implemented a CronJob described here (https://github.com/zalando/zalenium/issues/274#issuecomment-374215984) to clean up the file directories so `chown` does not take too long before pod healthcheck fails.

Keep in mind to use `cleanup?action=doCleanup` rather than `cleanup?action=doCleanupAll`
